resources/language/nl.ini

```diff
@@ -0,0 +1,489 @@
+language.code       = "nl"
+language.direction  = "ltr"
+language.isRTL = "false"
+language.dateformat = "d/m/Y"
+language.jsdateformat = "dd/mm/yy"
+language.timeformat = "h:i A"
+language.jstimeformat = "h:mm A"
```

Comment:
Same as above but in Javascript it's:
HH:mm

Reply:
Check! I'll adjust it